apps/design-system-docs/docusaurus.config.ts

```diff
@@ -0,0 +1,129 @@
+import { themes as prismThemes } from 'prism-react-renderer';
+import type * as Preset from '@docusaurus/preset-classic';
+import type { Config } from '@docusaurus/types';
+
+// This runs in Node.js - Don't use client-side code here (browser APIs, JSX...)
+
+const config: Config = {
+	title: 'Automattic Design System',
+	tagline: 'TODO: Do we have a tagline?',
+	// TODO: Add a favicon
+	favicon: 'img/favicon.ico',
+
+	// TODO: Set the production url here
+	url: 'https://your-docusaurus-site.example.com',
+	// Set the /<baseUrl>/ pathname under which your site is served
+	// For GitHub pages deployment, it is often '/<projectName>/'
+	baseUrl: '/',
+
+	onBrokenLinks: 'throw',
+	onBrokenMarkdownLinks: 'warn',
+
+	// Even if you don't use internationalization, you can use this field to set
+	// useful metadata like html lang. For example, if your site is Chinese, you
+	// may want to replace "en" with "zh-Hans".
+	i18n: {
+		defaultLocale: 'en',
+		locales: [ 'en' ],
+	},
+
+	presets: [
+		[
+			'classic',
+			{
+				docs: {
+					sidebarPath: './sidebars.ts',
+					// Remove this to remove the "edit this page" links.
+					editUrl: 'https://github.com/Automattic/wp-calypso/tree/trunk/apps/design-system-docs',
+					routeBasePath: '/',
+				},
+				theme: {
+					customCss: './src/css/custom.css',
+				},
+			} satisfies Preset.Options,
+		],
+	],
+
+	themeConfig: {
+		colorMode: {
+			disableSwitch: true,
+		},
+		// TODO: Replace with your project's social card
+		image: 'img/docusaurus-social-card.jpg',
```

Comment:
Let's AI-generate a social image to start with.

Reply:
Tracked in DS-149